### DISCUSSION TITLE
ElectronJS example

### DISCUSSION THREAD

**Discussion by dcmjtm** [tradingview/lightweight-charts#727] [General]
Is their a typescript example of lightweight-charts with electron JS?

**Comment by timocov** [comment]
@Nipheris is there any difference between ElectronJS (the web/view side) and the Browser?

For me it looks like you need to deal with lightweight-charts inside the web-part (not in nodejs one) and it should be pretty-similar to the way you'd work in the Browser. But at the moment we don't have such example (but perhaps somebody from the community can help with that).
